cobrand_hackney/static/_timeline.scss

```diff
@@ -31,15 +31,15 @@
     font-weight: 600;
 }
 
-.lbh-details {
+.lbh-timeline {
     .govuk-details__summary-text {
         font-size: 0.9em;
     }
     
     .govuk-details__text {
         background-color: lbh-colour("lbh-h04");
         border-left: none;
-        font-size: 0.9em;
+        font-size: 1em;
```

Comment:
This change isn't mentioned in the commit

Reply:
And isn’t really needed either! Thanks.